### DISCUSSION TITLE
Style inline quarto code

### DISCUSSION THREAD

**Discussion by arthurgailes** [quarto-dev/quarto-cli#5006] [Q&A]
Hello,

I'm drafting a report, and I'd like to bold all of my inline variables (e.g. `r some_variable`). Is there a way to do this via yml options?

Thank you!

**Comment by mcanouil** [comment]
It will depends on the format you are using, but for HTML, you can use CSS/SCSS to tweak how inline code is rendered or you can used spans (<https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>).
You can search through the documentation using keyword such as "styling" and/or "style" to find the above information and more.

If you want more, please provide a small but complete working example.

**Comment by arthurgailes** [comment]
Thanks for your answer. Here's a reproducible example if it helps. I'm trying to find a way to do this programatically - *not* wrapping the text in markdown or html tags. I have dozens of these in my code, so I'd like to be able to alter the styling in one shot.

```
---
format: docx
---

Here is the `r 1 + 0`st and `r 2 + 0`nd example.
```

**Comment by mcanouil** [comment]
Oh you picked the "worst" format to do whatever programmatically and/or easily.
Two options:
- use a Lua filter to tweak the output, see <https://quarto.org/docs/extensions/filters.html>.
- create your own Word template, see <https://quarto.org/docs/output-formats/ms-word-templates.html>.
